src/model/transferlog.js

```diff
@@ -102,21 +143,20 @@ module.exports = class TransferLog {
     return true
   }
 
-  * _storePackaged (packaged) {
-    this._storeInCache(packaged)
-    this._store.put('transfer_' + packaged.transfer.id, JSON.stringify(packaged))
+  _storePackaged (packaged) {
+    this._put(packaged.transfer.id, JSON.stringify(packaged))
```

Comment:
The `sendTransfer` and `handleSendTransfer` functions make sure a transfer isn't a duplicate before they do anything